docs/package-inclusion-guide.md

```diff
@@ -0,0 +1,55 @@
+# Azure Cloud Shell Package Inclusion Guide
+
+## Background
+
+There are millions (or potentially billions) of Linux packages. This guide goes through the principles of what the Azure Cloud Shell team uses to consider what gets included in the service.
```

Comment:
```suggestion
There are multiple Linux packages that exists out there, what determines which tool / package is installed in the Cloud Shell? This guide goes through the principles that help decide what gets included in the Cloud Shell service.
```

Reply:
I also think we could say there are a "large number of linux packages" - we just dont know if the number is millions or billions.